applications/mne_analyze/libs/anShared/Model/fiffrawviewmodel.cpp

```diff
@@ -436,6 +436,14 @@ FIFFLIB::FiffInfo* FiffRawViewModel::getFiffInfo() const
 
 //=============================================================================================================
 
+QSharedPointer<FIFFLIB::FiffInfo> FiffRawViewModel::getFiffInfo(bool bPlaceholder) const
```

Comment:
what is `bPlaceholder` for? Can we remove it?

Reply:
just remove the getFiffInfo() function which returns a raw pointer so that we only the one which returns the shared pointer